src/WebJobs.Script.SiteExtension/WebJobs.Script.SiteExtension.proj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <Import Project="../../build/common.props" />
+  <Import Project="../../build/workers.props" />
+
+  <PropertyGroup>
+    <!-- TFM and RId are actually used. -->
+    <TargetFramework>net8.0</TargetFramework>
+    <RuntimeIdentifier>win</RuntimeIdentifier>
+    <SiteExtensionName>Functions</SiteExtensionName>
+    <CustomAfterNoTargets Condition="'$(TargetFramework)' != ''">$(MSBuildThisFileDirectory)Publish.SingleTFM.targets</CustomAfterNoTargets>
+    <CustomAfterNoTargets Condition="'$(TargetFramework)' == ''">$(MSBuildThisFileDirectory)Publish.MultiTFM.targets</CustomAfterNoTargets>
+    <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- We don't ship python with the site extension. -->
+    <PackageReference Remove="Microsoft.Azure.Functions.PythonWorker" />
```

Comment:
Could this move to `python.props` conditional on the RID? Ultimately, Python is excluded from Windows builds of the host.

Reply:
@fabiocav I was under the impression it would be needed for local development of windows. I wanted it still included in the `WebHost.csproj`.

Reply:
Unless there's a scenario I'm missing, we should be OK as those flows won't specify a RID. We could use a different condition check if needed, but don't treat this as a blocker. Just thought it would be nice to have it all in that same file.

Reply:
Done. It _works_, but a bit hacky. For `SiteExtension.proj`, works fine as RID is set for both restore & build. For `WebHost.csproj`, RID is set only for build, so restore does incorporate the python worker targets still. But they then get removed from the `SuppressPythonWorker` target